cmd/system-probe/subcommands/run/command.go

```diff
@@ -341,6 +341,10 @@ func startSystemProbe(log log.Component, statsd compstatsd.Component, telemetry
 			if lcc := ddebpf.NewLockContentionCollector(); lcc != nil {
 				telemetry.RegisterCollector(lcc)
 			}
+			telemetry.RegisterCollector(ddebpf.NewLockContentionCollector())
+			if ec := ebpftelemetry.NewEBPFErrorsCollector(); ec != nil {
+				telemetry.RegisterCollector(ec)
+			}
```

Comment:
the lock contention collector is registered above
```suggestion
			if ec := ebpftelemetry.NewEBPFErrorsCollector(); ec != nil {
				telemetry.RegisterCollector(ec)
			}
```

Reply:
bad conflicts resolving :(